src/utils/dangerousStyleValue.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+import unitless from '@emotion/unitless';
+
+// Taken from https://github.com/facebook/react/blob/b87aabdfe1b7461e7331abb3601d9e6bb27544bc/packages/react-dom/src/shared/dangerousStyleValue.js
+export default function dangerousStyleValue(name: string, value: any): any {
+  // https://github.com/amilajack/eslint-plugin-flowtype-errors/issues/133
+  // $FlowFixMe
+  if (typeof value === 'symbol') {
+    return value;
+  }
+
+  const isEmpty = value == null || typeof value === 'boolean' || value === '';
+  if (isEmpty) {
+    return '';
+  }
+
+  if (
+    typeof value === 'number' &&
+    value !== 0 &&
```

Comment:
@probablyup unitless zero is part of CSS 2.1 which is supported even in IE 8.
As i know, React supports IE 9+, which makes adding px to 0 senseless.
This file is mostly copied "as it" from react source code. I think it will be strange when react itself doesn't apply px to zero, but styled-components will do it.

Reply:
Yes but it's not required so let's save the few bytes

Reply:
I don't want to be nitpicky, but a few bytes that we save here is putting more strain on gzip; either way were talking 6-7 chars after uglification. The bulk is still the map of properties 😅